src/_tutorials/web/images/client-server.svg

```diff
@@ -0,0 +1,3 @@
tMTkuMmMtMC44LDAtMC44LTAuOC0wLjgtMC44di02LjRjMC0wLjgsMC44LTAuOCwwLjgtMC44aDE5LjImIzEwOyYjOTsmIzk7YzAuOCwwLDAuOCwwLjgsMC44LDAuOFY0MDQuOHoiLz4mI3hhOwk8cGF0aCBzdHlsZT0iZmlsbDojMDAyMzNGOyIgZD0iTTQyNi40LDQxNmMwLDAuOCwwLDAuOC0wLjgsMC44aC0xOS4yYy0wLjgsMC0wLjgtMC44LTAuOC0wLjh2LTYuNGMwLTAuOCwwLjgtMC44LDAuOC0wLjhoMTkuMiYjMTA7JiM5OyYjOTtjMC44LDAsMC44LDAuOCwwLjgsMC44VjQxNnoiLz4mI3hhOzwvZz4mI3hhOzxwYXRoIHN0eWxlPSJmaWxsOiNDN0Q1REQ7IiBkPSJNMzAxLjYsNDgzLjJjMCwxLjYtMS42LDMuMi0zLjIsMy4yaC05Mi44Yy0xLjYsMC0zLjItMS42LTMuMi0zLjJ2LTQ4YzAtMS42LDEuNi0zLjIsMy4yLTMuMmg5MiYjMTA7JiM5O2MxLjYsMCwzLjIsMS42LDMuMiwzLjJ2NDhIMzAxLjZ6Ii8+JiN4YTs8cGF0aCBzdHlsZT0iZmlsbDojRDhFM0VBOyIgZD0iTTMwMS42LDQ4My4yYzAsMS42LTEuNiwzLjItMy4yLDMuMmgtOTIuOGMtMS42LDAtMy4yLTEuNi0zLjItMy4ydi00NGMwLTEuNiwxLjYtMy4yLDMuMi0zLjJoOTImIzEwOyYjOTtjMS42LDAsMy4yLDEuNiwzLjIsMy4ydjQ0SDMwMS42eiIvPiYjeGE7PGNpcmNsZSBzdHlsZT0iZmlsbDojRTdFRUYyOyIgY3g9IjQ0OCIgY3k9IjI2Ni40IiByPSI2LjQiLz4mI3hhOzxwYXRoIHN0eWxlPSJmaWxsOiMxRkVBREI7IiBkPSJNNDQ2LjQsMjI1LjZWMzguNGMwLTkuNiwwLTE3LjYtOS42LTE3LjZINjcuMmMtOS42LTAuOC05LjYsNy4yLTkuNiwxNy42djE4OGgzODguOFYyMjUuNnoiLz4mI3hhOzxwYXRoIHN0eWxlPSJmaWxsOiMxNUNDQ0M7IiBkPSJNNDQ2LjQsMjI1LjZWMzguNGMwLTkuNiwwLTE3LjYtOS42LTE3LjZINjcuMmMtOS42LTAuOC05LjYsNy4yLTkuNiwxNy42Ii8+JiN4YTs8cGF0aCBzdHlsZT0iZmlsbDojMUFCNUI1OyIgZD0iTTQzNi44LDIwSDY3LjJjLTkuNiwwLTkuNiw4LTkuNiwxOC40djE2LjhjMC05LjYsMC0xNy42LDkuNi0xNy42SDQzNmM5LjYsMCw5LjYsOCw5LjYsMTcuNlYzOC40JiMxMDsmIzk7QzQ0Ni40LDI4LDQ0Ni40LDIwLDQzNi44LDIweiIvPiYjeGE7PHBhdGggc3R5bGU9ImZpbGw6IzA4NDQ2RDsiIGQ9Ik00MTAuNCwyNjIuNGMwLDEuNi0xLjYsMi40LTQsMi40SDk3LjZjLTIuNCwwLTQtMC44LTQtMi40di0xMi44YzAtMS42LDEuNi0yLjQsNC0yLjRoMzA4LjgmIzEwOyYjOTtjMi40LDAsNCwwLjgsNCwyLjRWMjYyLjR6Ii8+JiN4YTs8cGF0aCBzdHlsZT0iZmlsbDojMDAyMzNGOyIgZD0iTTkzLjYsMjQ4LjhjMC0xLjYsMS42LTIuNCw0LTIuNGgzMDguOGMyLjQsMCw0LDAuOCw0LDIuNHYxMi44YzAsMS42LTEuNiwyLjQtNCwyLjQiLz4mI3hhOzxwb2x5Z29uIHN0eWxlPSJmaWxsOiNEOEUzRUE7IiBwb2ludHM9IjI1MiwyNDAuOCAyNTYuOCwyNDUuNiAyNjAsMjQyLjQgMjUyLDIzNC40IDI0NCwyNDIuNCAyNDcuMiwyNDUuNiAiLz4mI3hhOzxjaXJjbGUgc3R5bGU9ImZpbGw6IzI3RDY4QTsiIGN4PSIyNTIiIGN5PSIxMy42IiByPSIzLjIiLz4mI3hhOzxwYXRoIHN0eWxlPSJmaWxsOiMxNUFGODM7IiBkPSJNMjUyLDEwLjRjMS42LDAsMy4yLDEuNiwzLjIsMy4ycy0xLjYsMy4yLTMuMiwzLjIiLz4mI3hhOzxlbGxpcHNlIHN0eWxlPSJmaWxsOiNDRkZGRUY7IiBjeD0iMjUwLjM3MyIgY3k9IjEyLjc4MiIgcng9IjAuOCIgcnk9IjAuOCIvPiYjeGE7PGc+JiN4YTs8L2c+JiN4YTs8Zz4mI3hhOzwvZz4mI3hhOzxnPiYjeGE7PC9nPiYjeGE7PGc+JiN4YTs8L2c+JiN4YTs8Zz4mI3hhOzwvZz4mI3hhOzxnPiYjeGE7PC9nPiYjeGE7PGc+JiN4YTs8L2c+JiN4YTs8Zz4mI3hhOzwvZz4mI3hhOzxnPiYjeGE7PC9nPiYjeGE7PGc+JiN4YTs8L2c+JiN4YTs8Zz4mI3hhOzwvZz4mI3hhOzxnPiYjeGE7PC9nPiYjeGE7PGc+JiN4YTs8L2c+JiN4YTs8Zz4mI3hhOzwvZz4mI3hhOzxnPiYjeGE7PC9nPiYjeGE7PC9zdmc+" preserveAspectRatio="none"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
If we could make this one somehow take up a bit less vertical space that may be good.